krishnankit/shopping_cart/code/main.rb

```diff
@@ -0,0 +1,41 @@
+load "service.rb"
+products = []
+cart = []
+
+
+while 1
+  display_menu()
+  print "Your option: "
+  user_input = gets.chomp.to_i
+  puts "Mocked."
+  case user_input
+    when 0
+      break
+    when 1
+      display_products(products)
+    when 2
+      print "Enter the name: "
+      name = gets.chomp.downcase
+      print "Enter the price: "
+      price = gets.chomp.to_i
+      add_product(products, name, price)

```

Comment:
Inventory should also have quantity as an attribute and according to that it manages if inventory has enough quantity or not while making purchase

Reply:
Added quantity.